proto/api.proto

```diff
@@ -147,9 +147,7 @@ message GetResponse {
   optional Value value = 2;
 }
 
-// A PutRequest is arguments to the Put() method. Note that to write
-// an empty value, the value parameter is still specified, but Bytes
-// is set to nil.
+// A PutRequest is arguments to the Put() method.
```

Comment:
Just following the language used elsewhere in this file. Changed to `is the argument`.